python/ray/autoscaler/aws/example-full.yaml

```diff
@@ -16,7 +16,7 @@ upscaling_speed: 1.0
 # Empty string means disabled.
 docker:
     image: "rayproject/ray-ml:latest-gpu" # You can change this to latest-cpu if you don't need GPU support and want a faster startup
-    # image: rayproject/ray:latest-gpu   # use this one if you don't need ML dependencies, it's faster to pull
+    # image: rayproject/ray:latest-cpu   # use this one if you don't need ML dependencies, it's faster to pull
```

Comment:
I think this is meant to differentiate `ray` from `ray-ml`, not `cpu` from `gpu`.

Reply:
agreed, therefore they should be consistently using gpu image